sklearn/ensemble/gradient_boosting.py

```diff
@@ -404,7 +404,6 @@ class QuantileLossFunction(RegressionLossFunction):
 
     def __init__(self, n_classes, alpha=0.9):
         super(QuantileLossFunction, self).__init__(n_classes)
-        assert 0 < alpha < 1.0
```

Comment:
So this is not needed because it is checked before getting here?

Reply:
This is not needed because it is checked **after** getting here, with a better error message

Reply:
OK great.